<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FCashReportSummary</class>
 <widget class="QWidget" name="FCashReportSummary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>From</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnBack">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../Resort/res.qrc">
       <normaloff>:/images/left.png</normaloff>:/images/left.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EDateEdit" name="deStart"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>To</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EDateEdit" name="deEnd"/>
   </item>
   <item>
    <widget class="QToolButton" name="btnForward">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../Resort/res.qrc">
       <normaloff>:/images/right.png</normaloff>:/images/right.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chHotel">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string>Hotel</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chRestaurant">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string>Restaurant</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EDateEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">edateedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resort/res.qrc"/>
 </resources>
 <connections/>
</ui>
